server/routes/conversation.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createConversation,
+  getConversations,
+  createConversationViaProfile,
+} = require("../controllers/conversation");
+
+router.route("/").get(protect, getConversations); //GET a conversation
+router.route("/").post(protect, createConversation); //CREATE a conversation
+router.route("/profile").post(protect, createConversationViaProfile); //CREATE a conversation
```

Comment:
Nice one. But the comments are not needed.

Reply:
Comments removed